src/components/Loader/Loader.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { useContext } from 'react';
+import { Loader } from 'semantic-ui-react';
+// import { CircularProgress } from '@material-ui/core';
+import ThemeContext from '../../context/ThemeContext';
+
+const LoaderComponent = () => {
+  const { light } = useContext(ThemeContext);
```

Comment:
For booleans try to use prefixes like "hasLight, isLight" this way sounds more semantically correct plus, you give a hint to other devs to see that they are booleans :)